helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -66,6 +66,11 @@ rules:
     verbs:
       - create
       - delete
+      - get
+      - list
+      - patch
+      - update
+      - watch
```

Comment:
Do we need all those?

Reply:
We probably do not need list, but that should be it. I think we need to refactor some part of the project after this story

Reply:
I think we only need 
```
- create
- delete
- get
- update
```

Reply:
Lets try